SAKPaaS/src/app/components/occupancy-view/occupancy-view.component.html

```diff
@@ -1,13 +1,13 @@
 <ng-container *ngIf="isSearchBar; else isNoSearchBar">
   <div *ngIf="occupancy.value !== null">
-    <img [src]="iconPath" alt="Auslastung" />
+    <img [src]="iconPath" [attr.alt]="getText()" />
```

Comment:
Was macht das anders als beispielsweise `alt="{{ 'image-alt.occupancy' | translate }}"` bzw. warum an dieser Stelle anders als sonst?

Reply:
In der Funktion getText() wird bereits übersetzt

Reply:
@lepathie Es ist prinzipiell egal. Ich finde den _Property binding_-Ansatz (also `[alt]="getText()`) gegenüber dem von _String interpolation_ (`alt="{{ getText() }}"`) schöner und übersichtlicher. [Kann da den Guide von derr template syntax empfehlen](https://angular.io/guide/template-syntax) (speziell das Kapitel _property binding vs interpolation_ für den speziellen Fall).